lib/modules/hosts/imgur.js

```diff
@@ -219,7 +219,6 @@ const imgur = {
 			type: 'VIDEO',
 			controls: false,
 			href: ShowImages.module.options.preferredImgurLink.value === 'share' ? `${imgur.baseURL}${info.id}` : `${info.link}`,
-			downloadurl: info.download,
```

Comment:
I was suggesting something slightly different (using the advanced video controls); I think we can leave this as-is until we have a need for download URLs other than the first source (YAGNI).

`download` [is not in the imgur data model](https://api.imgur.com/models/image) so I don't think it's coming back.